test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -257,31 +261,113 @@ private static void getFailureReason() {
     }
 
     /**
-     * Position the instruction frame with testWindow (testcase created
-     * window) by the specified position.
-     * Note: This method should be invoked from the method that creates
-     * testWindow.
+     * Approximately positions the instruction frame relative to the test
+     * window as specified by the {@code position} parameter. If testWindow
+     * is null, only the instruction frame is positioned according to
+     * {@code position} parameter. This method should be called before making
+     * the test window visible.
+     *
+     * @param testWindow test window that the test is created.
+     *                   May be {@code null}.
+     *
+     * @param position  position must be one of:
+     *                  HORIZONTAL - the test instruction frame is positioned
+     *                  such that its right edge aligns with screen's horizontal
+     *                  center and the test window (if not null) is placed to
+     *                  the right of the instruction frame.
      *
-     * @param testWindow test window that the test is created
-     * @param position  position can be either HORIZONTAL (both test
-     *                  instruction frame and test window as arranged
-     *                  side by side) or VERTICAL (both test instruction
-     *                  frame and test window as arranged up and down)
+     *                  VERTICAL - the test instruction frame is positioned such
+     *                  that its bottom edge aligns with the screen's vertical
+     *                  center and the test window (if not null) is placed below
+     *                  the instruction frame.
+     *
+     *                  TOP_LEFT_CORNER - the test instruction frame is positioned
+     *                  such that its top left corner is at the top left corner of
+     *                  the screen and the test window (if not null) is placed to
+     *                  the right of the instruction frame.
      */
     public static void positionTestWindow(Window testWindow, Position position) {
         Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
+
+        // to get the screen insets inorder to position the frame by taking into
+        // account the location of taskbar/menubars on screen
```

Comment:
```suggestion
        // Get the screen insets in order to position the frame by taking into
        // account the location of taskbar/menubars on screen
```
Maybe drop _“in order”_?